src/main/java/frc/robot/commands/grabber/VisionAlignmentCommand.java

```diff
@@ -0,0 +1,67 @@
+package frc.robot.commands.grabber;
+
+import org.photonvision.PhotonCamera;
+import org.photonvision.targeting.PhotonPipelineResult;
+import org.photonvision.targeting.PhotonTrackedTarget;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+import frc.robot.subsystems.RollerSubsystem;
+import frc.robot.subsystems.RollerSubsystem.HeldPiece;
+import frc.robot.subsystems.drivetrain.BaseSwerveSubsystem;
+import frc.robot.vision.PhotonWrapper;
+
+/**
+ * Intakes 1 game piece with the roller mech. This command runs the rollers at a set power
+ * until the limit switch is triggered.
+ */
+public class VisionAlignmentCommand extends CommandBase {
+    private final BaseSwerveSubsystem driveSubsystem;
+
+    private double angularPower;
+    private final PIDController turnPID;
+    PhotonCamera recognitionCamera;
+
+    PhotonPipelineResult result;
+    boolean hasTargets;
+    PhotonTrackedTarget target;
+
+    public VisionAlignmentCommand(BaseSwerveSubsystem driveSubsystem){
+        this.driveSubsystem = driveSubsystem;
+
+        recognitionCamera = new PhotonCamera("Microsoft_LifeCam_HD-3000");
+        turnPID = new PIDController(0.2/35, 0.0, 0.0);
+        addRequirements(driveSubsystem);
+    }
+
+    @Override
+    public void initialize() {
+    } 
+
+    @Override
+    public void execute() {
+        result = recognitionCamera.getLatestResult();
+        hasTargets = result.hasTargets();
+        target = result.getBestTarget();
```

Comment:
What does this object look like if `hasTargets` is false? Does the command end execution at all if that happens?